File: ElectricityBoardFE/src/app/classes/applicant/applicant.ts
export class Applicant {
    id: number = 0;
    applicantName: string = "";
    gender: string = "";
    district: string = "";
    state: string = "";
    pincode: number = 0; 
    ownership: string = "";
    govtIdType: string = "";
    idNumber: number = 0; 
    category: string = "";
    loadApplied: number = 0;
    applicationDate: Date = new Date();
    approvalDate: Date = new Date();
    modifiedDate: Date = new Date();
    status: string = "";
    reviewerId: number = 0;
    reviewerName: string = "";
    reviewerComments: string = "";
}

File: ElectricityBoardBE/ElectricityBoardBE/src/main/java/com/srishti/electricityboardbe/entity/Applicant.java
package com.srishti.electricityboardbe.entity;

import lombok.Data;

import javax.persistence.*;
import java.time.LocalDate;

@Entity
@Data
public class Applicant {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "ID")
    private int id;

    @Column(name = "Applicant_Name")
    private String applicantName;

    @Column(name = "Gender")
    private String gender;

    @Column(name = "District")
    private String district;

    @Column(name = "State")
    private String state;

    @Column(name = "Pincode")
    private int pincode;

    @Column(name = "Ownership")
    private String ownership;

    @Column(name = "GovtID_Type")
    private String govtIdType;

    @Column(name = "ID_Number")
    private long idNumber;

    @Column(name = "Category")
    private String category;

    @Column(name = "\"Load_Applied (in KV)\"")
    private int loadApplied;

    @Column(name = "Date_of_Application")
    private LocalDate applicationDate;

    @Column(name = "Date_of_Approval")
    private LocalDate approvalDate;

    @Column(name = "Modified_Date")
    private LocalDate modifiedDate;

    @Column(name = "Status")
    private String status;

    @Column(name = "Reviewer_ID")
    private int reviewerId;

    @Column(name = "Reviewer_Name")
    private String reviewerName;

    @Column(name = "Reviewer_Comments")
    private String reviewerComments;
}
